src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <img
+        src="/images/imac.jpeg"
+        alt="mac"
+        class="card__img"
+      >
+
+      <h2 class="card__product-title">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__review">
+        <ul class="stars">
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__active-star"></li>
+
+          <li class="stars__unactive-star"></li>
```

Comment:
If you have repeated tags, each of which occupies one line, i.e. closes in the same line where it opened, then you can not create an empty line between such tags
```suggestion
        <ul class="stars">
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__active-star"></li>
          <li class="stars__unactive-star"></li>
```